src/Services/Settings.vala

```diff
@@ -30,5 +30,36 @@ public class Tooth.Settings : GLib.Settings {
 	void init (string key) {
 		bind (key, this, key, SettingsBindFlags.DEFAULT);
 	}
-
 }
+
+public enum Tooth.ColorScheme {
+	SYSTEM,
```

Comment:
This is an enumeration with the values that the application will handle. SYSTEM, LIGHT and DARK. It two methods:

* to_string (): Used to display it in the preferences window. They're translatable too!
* to_adwaita_scheme (): Which converts the values to their equivalent in Adw.ColorScheme